tests/conftest.py

```diff
@@ -39,14 +38,33 @@ def parallel_ray():
     # of return, and add parallel.shutdown() after the yield statement.
     # See https://docs.pytest.org/en/6.2.x/fixture.html#yield-fixtures-recommended
     # Currently, only a single ray setup is used, so this is not a problem.
+    "first we just check that the initialization works with some dummy ray_init parameters"
```

Comment:
If this is a comment, it would be better to use `#`

Reply:
ok